modules/nf-scil/bundle/recognize/meta.yml

```diff
@@ -0,0 +1,58 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/yaml-schema.json
+name: "bundle_recognize"
+description: Extract bundles from one (or more) tractograms
+keywords:
+  - Diffusion MRI
+  - Tractogram
+  - Segmentation
+tools:
+  - "Scilpy":
+      description: "The Sherbrooke Connectivity Imaging Lab (SCIL) Python dMRI processing toolbox."
+      homepage: "https://github.com/scilus/scilpy.git"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'test', single_end:false ]`
+
+  - tractograms:
+      type: file
+      description: Tractogram file(s) for bundle extraction
+      pattern: "*.trk"
+
+  - transform:
+      type: file
+      description: Transform matrix bringing the atlas to native space
+      pattern: "*.mat"
+
+  - config:
+      type: file
+      description: Bundle atlas configuration file
+      pattern: "*.json"
+
+  - directory:
+      type: directory
+      description: Directory containing the reference atlas
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'test', single_end:false ]`
+
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+
+  - bundles:
+      type: file
+      description: Segmented and cleaned bundles
+      pattern: "*.trk"
```

Comment:
Can you order those so that it matches the output of the main.nf? (meta, bundles, versions)

Reply:
Yessir here 1e22e91